ApartmentApp/app/views/apartments/_form.html.erb

```diff
@@ -1,10 +1,10 @@
-<%= form_for(apartment) do |f| %>
-  <% if apartment.errors.any? %>
+<%= form_for @apartment, html: { multipart: true } do |f| %>
```

Comment:
Added @ to apartment, seemed to work fine without it but just wanted to keep it consistent with the instance variable defined in ApartmentController